ledger/participant-integration-api/src/main/scala/platform/usermanagement/PersistentUserManagementStore.scala

```diff
@@ -89,21 +89,21 @@ class PersistentUserManagementStore(
 
   private val logger = ContextualizedLogger.get(getClass)
 
-  implicit private val loggingContext: LoggingContext = LoggingContext.newLoggingContext(identity)
+  private val defaltLoggingContext: LoggingContext = LoggingContext.newLoggingContext(identity)
```

Comment:
Why not wire the logging context from the ApiUserManagementService also for the read requests such as listUsers and getUserInfo?
The only case where a default context would have to be constructed is in the cachedUserManagement, where loading of data sometimes happens outside of the user reqest.

Reply:
Good point, applied